Makefile

```diff
@@ -172,6 +172,9 @@ test-util-pvc: generate manifests envtest
 test-drenv:
 	$(MAKE) -C test
 
+test-e2e:
+	go test ./e2e
```

Comment:
When running locally you want to test one or more tests against one or more environments. How are we passing the environment?

Maybe this is not needed from the makefile, maybe it should run all the tests, and for running specific test you will use go test directly. However this must be documented so everyone can use it.

Reply:
Can be nice to add something like:

```
test-e2e-binary:
    go test -o ramen-e2e .e2e
```

To make it easy to create a standalone ramen-e2e tool.